entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+case "$_DAPPNODE_GLOBAL_CONSENSUS_CLIENT_GNOSIS" in
+"gnosis-beacon-chain-prysm.dnp.dappnode.eth")
+  echo "Using gnosis-beacon-chain-prysm.dnp.dappnode.eth"
+  JWT_PATH="/security/prysm/jwtsecret.hex"
+  ;;
+"lighthouse-gnosis.dnp.dappnode.eth")
+  echo "Using lighthouse-gnosis.dnp.dappnode.eth"
+  JWT_PATH="/security/lighthouse/jwtsecret.hex"
+  ;;
+"teku-gnosis.dnp.dappnode.eth")
+  echo "Using teku-gnosis.dnp.dappnode.eth"
+  JWT_PATH="/security/teku/jwtsecret.hex"
+  ;;
+"nimbus-gnosis.dnp.dappnode.eth")
+  echo "Using nimbus-gnosis.dnp.dappnode.eth"
+  JWT_PATH="/security/nimbus/jwtsecret.hex"
+  ;;
+*)
```

Comment:
```suggestion
"lodestar-gnosis-dnp.dappnode.eth")
    echo "Using lodestar-gnosis-dnp.dappnode.eth"
    JWT_PATH="/security/lodestar/jwtsecret.hex"
    ;;
*)
```
Add lodestar switching logic so client doesnt need to be republished when lodestar is tested and ready.

Reply:
did in a PR to this branch #64, I cant add a new file to a review

Reply:
ill close this and put it in my PR as it should be organized sorry for the frustration